### ISSUE TITLE
Better linux compatibility

### ISSUE THREAD

**Issue by whyvn** [soler91/noita-together#91]
I'm really grateful for the app image but it seems like linux kind of an after thought (probably is). Like, there inst even an attempt to retrieve the game path in the updater (i was gonna fix this but i javascript is gross and i cant be sure it will work on both windows and linux without a few test machines running both). Most of the issues are with node though so i guess they have to fix some stuff. The last thing i wanted to say was that the wiki doesn't include any Linux stuff but I've made some changes but cant push them since I'm not a collaborator. Thank you!

also i think that
> if you are running linux (i dont know what im doing here) might need to run it with "--no-sandbox"

in the newest release should be taken out since it should be in the wiki and you shouldn't be running electron apps in root anyway.

**Comment by keyvaann** [comment]
I just downloaded the windows version and used "Add non-steam game to my library" to add the `Noita-Together-Setup-0.9.40.exe` to my library and then forced it to run with Proton and after that everything went on smoothly and I could join the game.

**Comment by whyvn** [comment]
i can use it just fine, thats not the problem (also why are you running under wine? its a linux native app since it uses electron just get the app image). the problem is the lack of information and etc. about plus it doesnt seam like soler can test it easily on linux judging by the comments he makes in the releases.

**Comment by keyvaann** [comment]
I tried the linux version and I had all kinds of problems and with searching in other issues here seems to be even more problems ahead if I wanted to continue that version so I tried the Windows version and it just worked.
Since I can't really contribute code wise I'm happy to use the windows version for now and let solar to focus so many other cool features that he can add to the mod like the new nt shared damage which is a blast to play.

**Comment by whyvn** [comment]
Yeah that's fine but it doesn't invalidate this issue. just because you don't have to use it doesn't mean the issues are irrelevant?

**Comment by 5HT2** [comment]
The appimage also gets stuck downloading assets after a few files. Re-tried a few times, never got past more than a dozen.

Added the exe version to steam as how @K1Hyve said, ran it through Proton Experimental and it installed + downloaded assets just fine. Should be pinned :+1:
